docs/tutorials/music-recognition-system.md

```diff
@@ -13,11 +13,11 @@ A music recognition system generally transforms audio data to embeddings and com
 
 ## Tutorial Overview
 
-Normally an audio embedding pipeline generates a set of embeddings given an audio path, which composes a unique fingerprint representing the input music. Each embedding corresponds to features extracted for a snippet of the input audio. By comparing embeddings of audio snippets, the system can determine the similarity between audios. The image below explains the music fingerprinting by audio embeddings.
+Normally an audio embedding pipeline generates a set of embeddings given an audio path, which composes a unique fingerprint representing the input music. The model selected in the pipeline reads the audio file as a tensor and splits it into shorter tensors in a fixed length. Then it takes the batch of tensors as input and returns an embedding for each small tensor. Each embedding corresponds to features extracted for a snippet of the input audio. By comparing embeddings of audio snippets, the system can determine the similarity between audios. The image below explains the music fingerprinting by audio embeddings.
 
 ![music_embedding](music_embedding.png)
 
-A block diagram for a basic music recognition system is shown in images below. The first image illustrates how the system transforms a music dataset to vectors with Towhee and then inserts all vectors into Milvus. The second image shows the querying process of an unknown music snippet.
+A block diagram for a basic music recognition system is shown in images below. The first image illustrates how the system transforms a music dataset to vectors with Towhee and then inserts all vectors into Milvus. The second image shows the querying process of an unknown music snippet. The system firstly uses the same pipeline for music dataset fingerprinting to generate a batch of vectors to represent the input music snippet. Then it retrieves the primary key of the closest vector in database for each vector in the batch. Matching the source music paths by primary keys of vectors, similarity search results vote for which one the query snippet is in the existing music dataset.
```

Comment:
A block diagram for a basic music recognition system is shown in the images below. The first image illustrates how the system transforms a music dataset into vectors with Towhee and then inserts all vectors into Milvus. The second image shows the querying process of an unknown music snippet. The system begins by using the same pipeline for music dataset fingerprinting to generate a batch of vectors to represent the input music snippet. With this batch it retrieves the primary key of the closest vector in the database for each vector in the batch. With these results, the audio file that has the most matched keys will be the closest match.